### PR TITLE
Adds a buffer between a limb's max damage and the brute required to sever

### PR DESCRIPTION
**Pull request by QualityVan**
## About The Pull Request
Just about the same brute damage as before on all limbs, but limbs can hold more damage than before so you need 20-30 burn damage to become immune to amputations instead of any at all.

## Why It's Good For The Game
Solves tactical 1 burn damage with minimal potential for increasing burn damage potency on the whole.

## Changelog
:cl:
balance: You need 20-30 burn damage on a limb in order for it to be immune to getting chopped off, instead of any at all.
/:cl:

### CODE REVIEW

**Comment by Lumipharon** [comment]
As mentioned in discord, this completely breaks HvH balance.

**Comment by Losenis** [comment]
You can get burn damage even if you're dead by being on a fire tile, this'll absolutely skyrocket the number of delimbs and make magmoor essentially impossible to play

**Comment by Serxule** [comment]
>  make magmoor essentially impossible to play

who takes magmoor in account for balance? its literally never chosen

**Comment by blackdav123** [comment]
> > make magmoor essentially impossible to play
> 
> who takes magmoor in account for balance? its literally never chosen

it is relevant because this would mean falling into lava and dying would result in all your limbs falling off in the goofiest way possible

better solution would be to allow burn damage to reach up to the damage cap and make any subsequent brute damage cause the delimb

**Comment by novaepee** [comment]
> > make magmoor essentially impossible to play
> 
> who takes magmoor in account for balance? its literally never chosen

Magmoor is gone.

**Comment by Losenis** [comment]
It only takes a map getting another "get burn damage if you step on it" hazard before the argument is back in play. Flamers already do burn damage to corpses by the fire tile anyway.

**Review by BraveMole — CHANGES_REQUESTED** [review]
Feels like the solution has vastly more impact than just fixing the bug.

**Comment by Serxule** [comment]
probably just make it so delimbing requires brute damage when damage is maxed and that fixes the problem (unless its already like that)

**Comment by Weedmaster77** [comment]
Don't like this idea at all it's just going make more marines get sent shipside for new limbs when it happens between 3 and 10 times a round on a pretty far non-steamroller round. Delimbing is powerful enough with brute damage burn doesn't need to make it even more punishing then it already is. An when ravs and warriors can just pop your arms, hands and legs off pretty easy with enough targeting which skilled xeno plays do pretty often. Also with light armor being the most commonly used armor delimbing is frequent enough . An with my many hours of un-suiting dead marines to rev I can confidently say that.

**Comment by Al-1ce** [comment]
Generally balancing your PR off of somebody falling into lava is a bad idea.

**Comment by Losenis** [comment]
It's not just lava, it's that tiles that do burn damage continue to do so even when the person is dead. See also: space, blue fire, WP smoke (probably), both combined in incen OB, some rare bug floating around that had corpses constnatly get tiny bits of damage that I never could pinpoint enough to bug report, etc. Lava just happens to be the thing most people already are familiar with, which is why it's the center of the argument. Magmoor wasn't the only one with map hazards either, there was that other map with the green toxin rivers that did the same thing without setting you on fire, as well as barrenquilla when it was in rotation. With brute damage it also happens but there aren't that many things out there to actually make it as pronounced, though one thing you can do in HvH is take a corpse's helmet off, help intent click it with knife, and you'd keep doing brute damage despite dead till you decap it.

The change clashes with too many things that may be bugs, oversights, intended mechanics, and so on for its own good.

**Comment by QualityVan** [comment]
Took the suggestion of requiring a brute hit in addition to reaching damage cap to cause a delimb. Should prevent the concerns raised about hvh and lava/fire.

**Comment by Losenis** [comment]
inb4 delimbed by a cigarette
image

**Review by BraveMole — CHANGES_REQUESTED** [review]
Like we talked in discord, I think I'd prefer having the delimb threshold be a percentage of max_damage. Let's say 80-90 %

**Comment by RikuTheKiller** [comment]
QV this overall makes it easier to delimb people.

I raised this point when the idea was originally brought up as well.

**Comment by RikuTheKiller** [comment]
Right now this makes arms easier to delimb. I'd recommend making it a flat 25 instead of a ratio so you can always just use that as a reference. (Take for example the arms, 150 isn't a 20% increase to 125, get your math right.)

**Comment by Losenis** [comment]
Is this actually necessary at all? Even a gamer that burns himself to avoid delimb just trades that to get killed faster assuming he actually burns the correct limb. Delimb is already extremely punishing because it means you get a robot limb that can't get healed by chems nor bandages/ointment and needs you to carry two more items they may not really have space for without a loadout change midround. At that point I personally just DNR and wait for the next round because even if it's possible to play like that it's not terribly fun.

**Comment by RikuTheKiller** [comment]
> Is actually necessary at all?

Yes, since you don't have to do it intentionally to benefit from it, just take some acid or flame damage and you're fully immune to delimbs, no intention to abuse this required.

**Comment by Losenis** [comment]
"benefit from it"

Bro you're dead

**Comment by Texicos** [comment]
Death is preferable to a fracture or delimb

**Comment by Losenis** [comment]
Not playing is preferable tbh

File: code/modules/organs/limbs.dm [hunk 1]
```diff
@@ -264,8 +266,8 @@
 		if(updating_health)
 			owner.updatehealth()
 		return update_icon()
-	if(CONFIG_GET(flag/limbs_can_break) && brute_dam >= max_damage * CONFIG_GET(number/organ_health_multiplier))
-		droplimb() //Reached max damage threshold through brute damage, that limb is going bye bye
+	if(CONFIG_GET(flag/limbs_can_break) && brute_dam >= max_damage * LIMB_MAX_DAMAGE_SEVER_RATIO * CONFIG_GET(number/organ_health_multiplier))
+		droplimb() //If the limb's at its max damage and takes any brute, it severs
```
Comment (QualityVan) [inline code/modules/organs/limbs.dm:270]:
```suggestion
		droplimb()
```

**Pull request merged**